src/platforms/java/guides/spring-boot/advanced-usage.mdx

```diff
@@ -40,6 +40,21 @@ public class CustomEventProcessor implements EventProcessor {
 }
 ```
 
+```kotlin
+import io.sentry.SentryEvent
+import io.sentry.EventProcessor
+import org.springframework.stereotype.Component
+
+@Component
+class CustomEventProcessor : EventProcessor {
+
+  override fun process(event: SentryEvent?, hint: Any?): SentryEvent? {
```

Comment:
we can improve this, `SentryEvent` should be non-null here

Reply:
https://github.com/getsentry/sentry-java/issues/1230

Reply:
Yes, but once we release new version.